scss/widgets/material/gridBase/_colors.scss

```diff
@@ -161,6 +161,9 @@ $datagrid-active-header-filter-icon-color: $base-accent !default;
 $datagrid-text-stub-background-image-path: null !default;
 $datagrid-filter-panel-color: $base-accent !default;
 
+$datagrid-text-link-disabled-opacity: 0.38 !default;
+$datagrid-icon-link-disabled-opacity: 0.6 !default;
```

Comment:
Did designer check these values?

Reply:
Yes, @DX-man told me to use same style as regular buttons have, so I copied them using devtools

Reply:
Instead of coping, you can use these contants directly:

```
$datagrid-text-link-disabled-opacity: $base-disabled-opacity;
$datagrid-icon-link-disabled-opacity: $button-disabled-icon-opacity;
```